treeherder/perf/fixtures/performance_bug_templates.yaml

```diff
@@ -12,6 +12,8 @@
 
       This is a list of all known regressions and improvements related to the push: {{ alertHref }}
```

Comment:
We should move this below {{ alertSummary }}. Also we should change this text to something like:

"You can find links to graphs and comparison views for each of the above tests at: {{ alertHref }}"